### ISSUE TITLE
Custom color range error for ordinal scales

### ISSUE THREAD

**Issue by eibanez** [vega/vega-lite#2373]
From the second example in the documentation of [custom color range](https://vega.github.io/vega-lite/docs/scale.html#example-custom-color-range):

> For ordinal, quantitative, and time fields, range can be a two-element array describing the two colors for interpolation.

However the example just cycles through the two colors.

**Comment by kanitw** [comment]
Can you provide an example spec that demonstrates the problem?  Thanks

**Comment by eibanez** [comment]
This example is in the page listed above"

```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "data": {"url": "data/cars.json"},
  "mark": "point",
  "encoding": {
    "x": {"field": "Horsepower", "type": "quantitative"},
    "y": {"field": "Miles_per_Gallon", "type": "quantitative"},
    "color": {
      "field": "Cylinders", "type": "ordinal",
      "scale": {"range": ["pink", "red"]}
    }
  }
}
```

**Comment by kanitw** [comment]
@eibanez oops sorry didn't catch that. Thanks for the report.

**Comment by kanitw** [comment]
Fixed in #2380
